components/climber.py

```diff
@@ -0,0 +1,31 @@
+class Climber:
+    def __init__(self) -> None:
+        self.deployed = False
+        self.retracted = False
```

Comment:
These variables are redundant. The `is_deployed` and `is_retracted` functions will determine this value every time we call it. They are not meant to serve as "getter" functions like in C++ or Java.